items/src/main/scala/uk/ac/wellcome/platform/api/items/Main.scala

```diff
@@ -25,9 +27,30 @@ object Main extends WellcomeTypesafeApp {
 
     val workService: StacksService = StacksServiceBuilder.build(config)
 
-    val router: ItemsApi = new ItemsApi {
+    val apiConf =
```

Comment:
Unless I've missed it elsewhere I think you'll need one of these https://github.com/wellcomecollection/catalogue-api/blob/fb6484df69a608c64b9555c6db8e8c71fcc8a0f3/search/src/main/scala/uk/ac/wellcome/platform/api/search/Main.scala#L29